packages/atlas/src/providers/nftActions/provider.tsx

```diff
@@ -17,35 +16,19 @@ type NftAction = 'putOnSale' | 'purchase' | 'settle'
 
 export const NftActionsProvider: React.FC = ({ children }) => {
   const [currentAction, setCurrentAction] = useState<NftAction | null>(null)
+  const [isBuyNowClicked, setIsBuyNowClicked] = useState<boolean>()
```

Comment:
Doesn't it produce any warning?
```suggestion
  const [isBuyNowClicked, setIsBuyNowClicked] = useState<boolean | undefined>()
```

Reply:
Nope, at least everything works for me.
![image](https://user-images.githubusercontent.com/51168865/159241244-fa76fda5-7b54-47da-9485-380d98222524.png)